react-ui/src/components/SubmissionRow.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * Entry for mod review and approval
```

Comment:
This has been overloaded to show submissions to mods for review and approval and to users to see their own submission statuses (and in the future withdraw, resubmit, and edit).

Reply:
Noted